dev-lang/ghc/ghc-9.2.7-r1.ebuild

```diff
@@ -34,20 +34,25 @@ ghc_binaries() {
 	local host="https://eidetic.codes"
 	echo "
 		binary? ( ${host}/${PN}-bin-${BIN_PVR}-${1}.gpkg.tar )
-		!binary? ( ${host}/${PN}-bin-${BUILD_BIN_PVR}-${1}.gpkg.tar )
 	"
+	[[ "${1}" == *-pc-* ]] && \
+		echo "!binary? ( ${host}/${PN}-bin-${BUILD_BIN_PVR}-${1}.gpkg.tar )" || \
+		echo "!binary? ( ${host}/${PN}-bin-${BUILD_BIN_PVR}-${1}.tar.gz )"
```

Comment:
It might also be relatively easy to repackage the `.tar.gz` files as `.gpkg.tar` so that everything is uniform.

Reply:
> It might also be relatively easy to repackage the `.tar.gz` files as `.gpkg.tar` so that everything is uniform.

Unfortunately if I knew how to do this I would have done it.  How would you prefer this to be laid out?  The problem is that the 9.0.2 binaries are in .tar.gz form, but the 9.2.7 binaries are in .gpkg.tar form.  Can you write out just a quick example?

Reply:
Ok, no problem. It looks like a proper `gpkg` file also has metadata from `/var/db/pkg`, so repackaging won't be an easy option.